src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1266,37 +1264,33 @@ function tryRemoveElement(elem) {
             const fallbackFontList = apiClient.getUrl('/FallbackFont/Fonts', {
                 api_key: apiClient.accessToken()
             });
-            const htmlVideoPlayer = this;
             const options = {
                 video: videoElement,
                 subUrl: getTextTrackUrl(track, item),
                 fonts: avaliableFonts,
-                workerUrl: `${appRouter.baseUrl()}/libraries/subtitles-octopus-worker.js`,
-                legacyWorkerUrl: `${appRouter.baseUrl()}/libraries/subtitles-octopus-worker-legacy.js`,
-                onError() {
-                    // HACK: Clear JavascriptSubtitlesOctopus: it gets disposed when an error occurs
-                    htmlVideoPlayer.#currentSubtitlesOctopus = null;
-
-                    // HACK: Give JavascriptSubtitlesOctopus time to dispose itself
-                    setTimeout(() => {
-                        onErrorInternal(htmlVideoPlayer, 'mediadecodeerror');
-                    }, 0);
-                },
+                fallbackFont: 'liberation sans',
+                availableFonts: {'liberation sans': `${appRouter.baseUrl()}/libraries/default.woff2`},
+                // eslint-disable-next-line compat/compat
+                workerUrl: `${appRouter.baseUrl()}/${new URL('jassub/dist/jassub-worker.js', import.meta.url)}`,
+                // eslint-disable-next-line compat/compat
+                legacyWorkerUrl: `${appRouter.baseUrl()}/${new URL('jassub/dist/jassub-worker-legacy.js', import.meta.url)}`,
```

Comment:
This results in incorrect URLs, such as `http://192.168.1.17:8096/web/http://192.168.1.17:8096/web/bd8c93ca1a6e55cd85f3.js`.
Reproducible in Chrome.

Reply:
ah shite, I expected webpack to compile the URL away to a string like it used to, I guess something changed

Reply:
I fixed this by using pathname, in theory it should be fine to just use the new URL standalone, but I decided not to as the baseUrl method might be used for cases which I'm not aware of so I kept it in

Reply:
you said webos 1.2, that's so ancient it doesn't have let and const, the non legacy is transpiled for consistency sake

Reply:
i don't really know why the URL is incorrect then, I'm gonna remove baseurl and you try that, cuz the current code works on chrome

Reply:
> you said webos 1.2, that's so ancient it doesn't have let and const, the non legacy is transpiled for consistency sake

I mean, it should already be prepared for legacy browsers. Emscripten devs discussed transpilation, iirc, but I don't remember their decision.
`let/const` from [Chrome 49](https://caniuse.com/const).
WASM from [Chrome 57](https://caniuse.com/wasm).
So, at least for Chrome, we can allow `let/const` for the non-legacy worker.

> I'm gonna remove baseurl

It seems to work.

Reply:
I compile the 2orkwr differently, and also use modern syntax, so I'm keeping it as it is

great, anything else to address or is this now mergable?